aicsimageio/transforms.py

```diff
@@ -16,6 +16,22 @@
 ###############################################################################
 
 
+def convert_list_to_slice(L: List) -> Union[int, List, slice]:
```

Comment:
I am tempted to say that we should also allow `tuple` in this attempt. In our function, tuples have the same functionality as lists https://github.com/AllenCellModeling/aicsimageio/blob/main/aicsimageio/transforms.py#L90

Reply:
Does the same bug happen with tuples?

Reply:
@colobas can you quick try the bug with a tuple instead of a list?

Reply:
oh - we actually convert dim_spec tuples to lists before this function gets called.
I can convert the function to accept tuples too, but it will never be called in this context.

Reply:
Sorry I missed your tag @evamaxfield !
Thanks for figuring this out!!